Geek-Collectors-Network/geek-collectors-network
# feat(frontend): implement client side routing

### Summary
Implement client-side routing using [React Router](https://reactrouter.com/en/main) to enable user navigation.

### Testing
Manually test routing:

1. Navigate to `./apps/frontend`.
2. Run `npm run start:dev`.
3. Open a web browser and access `http://localhost:8080/` to see the landing page.
4. Click `LOGIN` button and verify that you are navigated to the registration page.
5. Test the back button functionality to ensure redirection to the previous page.
